setup.py

```diff
@@ -0,0 +1,20 @@
+#!/bin/env python
+from setuptools import setup,find_packages
+
+setup(
+    name="pomodorotasks",
+    packages=find_packages(exclude=['test' 'test/*']),
+    install_requires=['tasklib', 'pygobject', 'dbus-python'],
+    entry_points={
+        'console_scripts': [
+            'pomodorotasks = pomodoro.client:main'
+            ],
+        'gui_scripts': [
+            'pomodoro-taskd = pomodorotasks.daemon:main',
+            ]
```

Comment:
What a mess of UX. xD

If I understand properly, pomodoro-taskd will be the final binary name for the gui script,right?

And  'pomodorotasks = pomodoro.client:main' will be changed to pomodoro-tasks,  why not using the current name pomodoro-client and pomodoro-daemon? ;)

I think I will use the setuptool installation too. :)